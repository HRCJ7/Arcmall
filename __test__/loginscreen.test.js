import "react-native";
import React from "react";
import renderer from "react-test-renderer";
  
it("renders the Image component", done => {
    const Image = require("Image");
    Image.getSize("logo.png", (width, height) => {
      const tree = renderer.create(<Image style={{ height, width }} />).toJSON();
      expect(tree).toMatchSnapshot();
      done();
    });
});

it("renders the Text component", () => {
    const Text = require("Text");
    const tree = renderer
      .create(<Text/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
});