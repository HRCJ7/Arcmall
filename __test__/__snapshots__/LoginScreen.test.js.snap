// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Image component 1`] = `
<Image
  style={
    Object {
      "height": 240,
      "width": 320,
    }
  }
/>
`;

exports[`renders the Text component 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
/>
`;
